src/components/ScenarioParameters/components/ScenarioParametersInputs/GenericSliderInput.js

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) Cosmo Tech.
+// Licensed under the MIT license.
+
+import React from 'react';
+import { BasicSliderInput } from '@cosmotech/ui';
+import PropTypes from 'prop-types';
+import { useTranslation } from 'react-i18next';
+import { TranslationUtils } from '../../../../utils';
+
+const DEFAULT_MIN_VALUE = 0;
+const DEFAULT_MAX_VALUE = 100;
+function getMinValue(parameterData) {
+  if (parameterData.minValue == null) {
+    return DEFAULT_MIN_VALUE;
+  }
+  return parameterData.minValue;
+}
+
+function getMaxValue(parameterData) {
+  if (parameterData.maxValue == null) {
+    return DEFAULT_MAX_VALUE;
+  }
+  return parameterData.maxValue;
+}
+
+export const GenericSliderInput = ({ parameterData, parametersState, setParametersState, context }) => {
+  const min = getMinValue(parameterData);
+  const max = getMaxValue(parameterData);
+  const { t } = useTranslation();
+  const setValue = (newValue) => {
+    setParametersState((currentParametersState) => ({
+      ...currentParametersState,
+      [parameterData.id]: newValue,
+    }));
+  };
+
+  let value = parametersState[parameterData.id];
+  if (value == null) {
+    value = NaN;
+  }
+  return (
+    <BasicSliderInput
+      key={parameterData.id}
+      data-cy={parameterData.dataCy}
```

Comment:
Note: `parameterData.dataCy` will no longer be used after merging the multi-workspaces branch, the `data-cy` tag value will be generated based on the parameter id and the input type (i.e. "slider" type)